magister-calendar.js

```diff
@@ -409,10 +468,16 @@ function parseAppointments(appointments, currentcourse) {
       appointment.prefix = "";
     }
 
+    // Make the title a bit more pretty
+    var title = appointment.description.split("-")[0].trim();
```

Comment:
Not all schools have a delimiter of `-` for all their titles ([see these examples](https://github.com/lesander/magister-calendar/blob/master/assets/examples.png)).